content/about/faq.md

```diff
@@ -68,121 +68,164 @@ Here are some Frequently Asked Questions about Haiku. For development related to
+However, there are ways to enjoy the web without Flash, such as using HTML5 compatible versions of webpages (youtube and grooveshark support this for
+example). Besides, Adobe [deprecated Flash](https://theblog.adobe.com/adobe-flash-update/) anyway.
 
 ### Is there an office suite?
 There are several options:
 
   * You can install [LibreOffice](https://www.libreoffice.org/) available at _HaikuDepot_.
   * You can find the [Caligra](https://www.calligra.org/) office suite at _HaikuDepot_.
-  * You can use an online tool such as [Google Docs](https://docs.google.com).
+  * You can use online tools, such as [Google Docs](https://docs.google.com).
   * You can use native applications, such as Sum-It (spreadsheet), WonderBrush (drawing and graphical design).
-  * You can run GoBe productive, the office suite for BeOS, if you have a copy of it. Unfortunately, it is not distributed anymore.
+  * If you happen to own an old copy of GoBe productive laying around, you can use it on Haiku with the 32-bit version of Haiku.
+    Unfortunately, it is not being distributed anymore.
 
 ### Are there any games for Haiku?
-Yes! _HaikuDepot_ offers a selection of open source and freeware games. Most games written for BeOS will also run fine, and you can use emulators such as DOSBox to run games originally written for other systems as well.
+Yes! _HaikuDepot_ offers a selection of open source and freeware games. Most games written for BeOS will also run fine,
+and you can use emulators, such as DOSBox, Dolphin, snes9x MAME and RetroArch, to run games originally written for
+other systems and consoles as well.
 
-### Is there python, ruby, perl?
+### Are Python, Ruby, Perl and Go available on Haiku?
 Yes! Haiku includes these common scripting languages and many more.
 
 ### Will GoBe Productive work in Haiku?
-Yes! Haiku's 32-bit installation is binary-compatible with BeOS R5, which means applications designed for BeOS R5 will run. There may still be a few drawing bugs and other glitches, remember Haiku is still in beta stage. But overall GoBe Productive should be usable.
+Yes! Haiku's 32-bit installation is binary-compatible with BeOS R5, which means applications designed for BeOS R5 will run.
+There may still be a few drawing bugs and other glitches, remember Haiku is still in beta stage.
+Overall, GoBe Productive should be usable.
 
 ## Releases
 ### When is the next release?
 You can check the status of the releases at the [roadmap page](https://dev.haiku-os.org/roadmap).
 
 ## Technology
 ### I heard Haiku still uses gcc2, is that true?
-Our own internal fork of gcc2 is used to compile the x86 32-bit release to maintain BeOS binary compatibility. While gcc2 is the primary compiler for the x86 32-bit release, it also includes a modern gcc7 which can be leveraged to compile newer applications requiring it.
+Our own internal fork of gcc2 is used to compile the x86 32-bit release to maintain BeOS binary compatibility. While gcc2 is the primary compiler
```

Comment:
now you have a repetition of "newer", which I guess is why "modern" was used previously